tfhe/src/core_crypto/entities/fourier_lwe_bootstrap_key.rs

```diff
@@ -363,75 +338,3 @@ impl<'a> FourierLweBootstrapKeyView<'a> {
         );
     }
 }
-
-impl<Scalar> FourierBootstrapKey<Scalar> for FourierLweBootstrapKeyOwned
```

Comment:
Any reason to remove this implementation ?

Reply:
that might have been a mistake on my part